utils/robotLogsHelper.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
```

Comment:
Not sure if this needs to be a separate file. Why not include it as just a method in the file where it is used?
Additionally I think you could potentially rework the troublesome while loop which this by just using the errors and retrying again then, else breaking on receiving the data correctly

Reply:
If we can evade the loop, we can avoid putting this snippet in a separate file.

Reply:
Is deleted now.